test/language/expressions/arrow-function/arrow/binding-test.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: this binding test
+es6id: 14.2
+---*/
+
+var obj = {
+    binder: function(){return () => this; }
+ }
+
+assert.sameValue(obj.binder()(), obj, "This binding test failed for obj.binder()().");
+
+var test = {
+    check1:obj.binder(),
+    check2:obj.binder
+ }
+
+assert.sameValue(test.check1(), obj, "test.check1() should return obj.");
+
+assert.sameValue(test.check2()(), test, "test.check2()() should return test.");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js